app/gui/qt/mainwindow.cpp

```diff
@@ -1262,6 +1262,13 @@ void MainWindow::load()
     delete load_from_dialog;
 }
 
+
+void MainWindow::launch_share(const QString share_filename)
```

Comment:
Can we make this function more generic so that its other uses are clearer - call it something like `open_save()`?